src/components/List/ListItem.module.css

```
@@ -1,9 +1,36 @@
 .list-item {
-  --component-list_item-border_bottom_color: #bcc7cc;
-  --component-list_item-border-bottom-style: solid;
-  --component-list_item-border-bottom-width: 0.1rem;
+  display: block;
   border-bottom-color: var(--component-list_item-border_bottom_color);
   border-bottom-style: var(--component-list_item-border-bottom-style);
   border-bottom-width: var(--component-list_item-border-bottom-width);
-  display: block;
+}
+
+.list-item--solid {
+  --component-list_item-border_bottom_color-solid: #bcc7cc;
+  --component-list_item-border-bottom-style-solid: solid;
+  --component-list_item-border-bottom-width-solid: 0.1rem;
+  --component-list_item-border_bottom_color: var(
+    --component-list_item-border_bottom_color-solid
+  );
+  --component-list_item-border-bottom-style: var(
+    --component-list_item-border-bottom-style-solid
+  );
+  --component-list_item-border-bottom-width: var(
+    --component-list_item-border-bottom-width-solid
+  );
+}
+
+.list-item--dashed {
+  --component-list_item_dashed-border-bottom-width-dashed: 0.1rem;
+  --component-list_item_dashed-border_bottom_color-dashed: #1eadf7;
+  --component-list_item_dashed-border_bottom_style-dashed: dashed;
+  --component-list_item-border_bottom_color: var(
+    --component-list_item_dashed-border_bottom_color-dashed
+  );
+  --component-list_item-border-bottom-style: var(
+    --component-list_item_dashed-border_bottom_style-dashed
+  );
+  --component-list_item-border-bottom-width: var(
+    --component-list_item_dashed-border-bottom-width-dashed
+  );
```

This should also be simplified:

```suggestion
.list-item--solid {
  --component-list_item-border_bottom_color: #bcc7cc;
  --component-list_item-border-bottom-style: solid;
  --component-list_item-border-bottom-width: 0.1rem;
}

.list-item--dashed {
  --component-list_item-border_bottom_color: #1eadf7;
  --component-list_item-border-bottom-style: dashed;
  --component-list_item-border-bottom-width: 0.1rem;
```